backend/api/blockchain/dependencies.py

```
@@ -0,0 +1,51 @@
+logger = logging.getLogger(__name__)
+
+
+async def get_health_manager(request: Request) -> ServiceHealthManager:
+    """Dependency to get the health manager from the app state."""
+    # This assumes the health_manager is attached to the app state
+    # In server.py, typically as app.state.health_manager
+    from backend.server import health_manager
+    return health_manager
+
+
+async def get_blockchain_service(health_manager: ServiceHealthManager = Depends(get_health_manager)) -> Any:
```

Ruff (0.12.2)

Line too long (108 > 100)

(E501)